src/actions/actionTypes.js

```diff
@@ -0,0 +1,11 @@
+import keyMirror from 'keymirror';
+
+export const ACTIONS = keyMirror({
```

Comment:
#### Critical
- Not the kinda selector generator I was looking for, it's not supposed to exceed more than 5 lines of code.